backend/api_calls.py

```diff
@@ -1055,6 +1056,29 @@ def MarkNotificationSeen(request, firebase):
       request['playerId'], request['notificationId']), put_data)
 
 
+def RegisterUserDevice(request, firebase):
+  """Register a user device to a userId.
+
+  Validation:
+    userId must exist.
+
+  Args:
+    userId: User id to associate with.
+    deviceToken: Ionic device token.
```

Comment:
That sounds reasonable.

If we do end up adding a "UpdateUser", we can change this method to call UpdateUser() and pass in just the device token.